components/list/list.css

```diff
@@ -0,0 +1,56 @@
+@import "../utilities.css";
+
+.container__components {
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.unordered-list-circle {
+  list-style-type: circle;
+}
+
+.unordered-list-disc {
+  list-style-type: disc;
+}
+
+.unordered-list-square {
+  list-style-type: square;
+}
+
+.ordered-list-lower-alpha {
+  list-style-type: lower-alpha;
+}
+
+.ordered-list-upper-alpha {
+  list-style-type: upper-alpha;
+}
+
+.ordered-list-upper-roman {
+  list-style-type: upper-roman;
+}
+.ordered-list-lower-roman {
+  list-style-type: lower-roman;
+}
+
+.stacked {
+  border: 1px solid black;
+  /* padding: 1rem; */
```

Comment:
Is this required ?

Reply:
Yup, not required, the stack class is needed but a few styles in it can be removed. Thank you @samyukthasudhakar for pointing it out.